### PR TITLE
Expose --storage.tsdb.retention.time in metric prometheus_tsdb_retention_limit_seconds

### PR DESCRIPTION
**Pull request by msscaroso**
This PR exposes the config option `--storage.tsdb.retention.time` as a metric.

It's assumed that tsdb.Option is an int64 representing time in milliseconds, based on its definition found here:
https://github.com/prometheus/prometheus/blob/624b973ebf4d0d7ed863b5538d7940c9c66c5cd0/cmd/prometheus/main.go#L1572

Related Issue: https://github.com/prometheus/prometheus/issues/12925

### CODE REVIEW

**Comment by bboreham** [comment]
Thanks for the PR!

Given there is already a metric `prometheus_tsdb_retention_limit_bytes`, would you consider naming this one `prometheus_tsdb_retention_limit_seconds` ?

**Comment by msscaroso** [comment]
Thank you for reviewing the PR @bboreham.
Makes sense. `prometheus_tsdb_retention_limit_seconds` looks better!

**Review by bboreham — APPROVED** [review]
LGTM; I'll wait for a bit before merging to see if any other maintainers have opinions.

**Comment by beorn7** [comment]
Message from the bug scrub: Apparently, no other maintainers have opinions. Merging.

**Pull request merged**
